server/app.js

```diff
@@ -6,7 +6,7 @@ Habitat.load();
 var env = new Habitat();
 
 // App
-var app = require('./server')(env);
+var app = require('./server')('sprinter.sqlite');
```

Comment:
No need of the argument, as it's already hard-coded in `server.js`
Please verify the changes minutely (setup a fresh sandbox staging, if required) to see if everything works.